src/Goerli-DssSpell.sol

```diff
@@ -46,19 +134,438 @@ contract DssSpellAction is DssAction {
+            POOL:            0x3354493615a21D544974e6665dc1851c6A117F9D, // Tinlake Operator
+            TRANCHE:         0x47335Eb13a12C126272a04E48eb09FC989135de3, // Tinlake Tranche
+            gemID:           "RWA012",
+            joinID:          "MCD_JOIN_RWA012_A",
+            urnID:           "RWA012_A_URN",
+            inputConduitID:  "RWA012_A_INPUT_CONDUIT",
+            outputConduitID: "RWA012_A_OUTPUT_CONDUIT",
+            pipID:           "PIP_RWA012",
+            ilk:             "RWA012-A",
+            ilkString:       "RWA012",
+            ilkRegistryName: "RWA012-A: Centrifuge: BlockTower Credit (III)",
+            RATE:            FOUR_PCT_RATE,
+            CEIL:            30_000_000,
+            PRICE:           36_499_587 * WAD,
+            MAT:             100_00, // Liquidation ratio
+            TAU:             0,      // Remediation period
+            DOC:             RWA010_A_DOC
+        }));
+
+
+        // RWA-013 Onboarding
+        // https://vote.makerdao.com/polling/QmNucsGt
+        _addCentrifugeCollateral(CentrifugeCollateralValues({
+            GEM_JOIN:        MCD_JOIN_RWA013_A,
+            GEM:             RWA013,
+            URN:             RWA013_A_URN,
+            OPERATOR:        RWA013_A_OPERATOR,
+            INPUT_CONDUIT:   RWA013_A_INPUT_CONDUIT,
+            OUTPUT_CONDUIT:  RWA013_A_OUTPUT_CONDUIT,
+            DROP:            0x0691FAEa2Eb8eBB2C36Fc24d577cA73AfbDB7Bdd,
+            OWNER:           0x116b030167Cc8A82C158f0598f4C4677f575Cc50, // Tinlake Clerk
+            POOL:            0x3C05eFC8D0fC042c5686b3989bbDb1E1D29dAec7, // Tinlake Operator
+            TRANCHE:         0x6A635Ada2eC663B9b38ca7a3E5c918D5D0B0E99D, // Tinlake Tranche
+            gemID:           "RWA013",
+            joinID:          "MCD_JOIN_RWA013_A",
+            urnID:           "RWA013_A_URN",
+            inputConduitID:  "RWA013_A_INPUT_CONDUIT",
+            outputConduitID: "RWA013_A_OUTPUT_CONDUIT",
+            pipID:           "PIP_RWA013",
+            ilk:             "RWA013-A",
+            ilkString:       "RWA013",
+            ilkRegistryName: "RWA013-A: Centrifuge: BlockTower Credit (IV)",
+            RATE:            FOUR_PCT_RATE,
+            CEIL:            70_000_000,
+            PRICE:           85_165_703 * WAD,
+            MAT:             100_00, // Liquidation ratio
+            TAU:             0,      // Remediation period
+            DOC:             RWA013_A_DOC
+        }));
+
+
+        // ----------------------------- Collateral offboarding -----------------------------
+        //  Offboard RENBTC-A
+        //  Poll Link:   https://vote.makerdao.com/polling/QmTNMDfb#poll-detail
+        //  Forum Post:  https://forum.makerdao.com/t/renbtc-a-proposed-offboarding-parameters-context/18864
+
+        DssExecLib.setIlkLiquidationPenalty("RENBTC-A", 0);
+        DssExecLib.setKeeperIncentiveFlatRate("RENBTC-A", 0);
```

Comment:
I think we need to set the chip to 0 as well.  Looking at etherscan it is currently: `1000000000000000` (https://goerli.etherscan.io/address/0xfeff5d71d665a4c0712cd87d802380958b7eb333#readContract)

```suggestion
        DssExecLib.setKeeperIncentiveFlatRate("RENBTC-A", 0);
        DssExecLib.setKeeperIncentivePercent("RENBTC-A", 0);
```